src/app/containers/CpsMetadata/utils/getBrandedImage/index.test.js

```diff
@@ -0,0 +1,44 @@
+import getBrandedImage from '.';
+
+describe('CpsMetadata get branded image', () => {
+  const ENV = process.env;
+  afterEach(() => {
+    process.env = ENV;
+  });
```

Comment:
You could do this instead of storing the ENV variable:

```
afterAll(() => {
    delete process.env.SIMORGH_APP_ENV;
});
```

Reply:
I am uncomfortable with the idea of leaving the test state without an important variable at the end the tests. Can you explain further why this is better?

Reply:
I had found that `process.env.SIMORGH_APP_ENV` was always `undefined`.  I found this code, which is why I changed my implementation to match.  https://github.com/bbc/simorgh/blob/a78ac1c54ea66f76c5d62e8fde88126ace8578f8/src/server/Document/index.test.jsx#L64 

But this is OK to leave it as is 😄

Reply:
Sounds good. I'll update it.